app_demo.rb

```diff
@@ -0,0 +1,81 @@
+require './lib/order'
+require './lib/submitorder'
+
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1, :total, :to_mobile, :from_mobile, :message_body, :message, :account_sid, :auth_token, :submitorder1, :quantity
```

Comment:
Metrics/LineLength: Line is too long. [171/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)